packages/financial-templates-lib/src/clients/OptimisticOracleClient.js

```diff
@@ -1,227 +0,0 @@
-// A thick client for getting information about an OptimisticOracle. Used to get price requests and
```

Comment:
why is this marked as deleted? it did this in a few places when you had just moved/re-named the file. I guess the diff was too large for it to see that it was from a previous file.

Reply:
Yeah, exactly.